_wallets/edgewallet.md

```diff
@@ -0,0 +1,49 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+id: edgewallet
+title: "Edge Mobile Wallet"
+titleshort: "Edge"
+compat: "mobile android ios"
+level: 3
+platform:
+  - mobile:
+    name: mobile
+    os:
+      - name: android
+        text: "walletedgewallet"
+        link: "https://play.google.com/store/apps/details?id=co.edgesecure.app"
+        source: "https://github.com/EdgeApp"
```

Comment:
This GitHub link needs to point to the repository that contains the source code for the Android app.

Reply:
This is now fixed